Cargo.toml

```diff
@@ -21,7 +21,7 @@ cfg-if = "1.0.0"
 ffi_helpers = "0.3.0"
 num_cpus = "1.15.0"
 wasm-bindgen = "=0.2.83" # Only for compatability with moose wasm version
-h2 = "=0.3.24" # RUSTSEC-2024-0003
+h2 = "=0.3.26" # RUSTSEC-2024-0332
```

Comment:
that is unrelated change, already being reviewed in #464

Reply:
So you would now recommend everyone to wait for #464 to be merged and block any other PRs? I don't see a point. For me it should be whoever merges it first. It's in a separate commit with a description. I don't see why it must also be in a separate PR.

Reply:
I don't think that this is urgent enough to make a mess.